src/ResourceManager/Compute/Commands.Compute/Generated/Disk/DiskCreateOrUpdateMethod.cs

```diff
@@ -62,7 +62,7 @@ protected object CreateDiskCreateOrUpdateDynamicParameters()
 
             var pDisk = new RuntimeDefinedParameter();
             pDisk.Name = "Disk";
-            pDisk.ParameterType = typeof(PSDisk);
```

Comment:
No.  The real change is in ProcessRecord() method.  The method, CreateDiskCreateOrUpdateDynamicParameters() and ExecuteDiskCreateOrUpdateMethod() are for the generalized cmdlet (Invoke cmdlets), but we no longer generated the cmdlets.   So anything in these methods do not have any effect on cmdlets.